packages/vue/src/components/atoms/SfInput/SfInput.vue

```diff
@@ -24,6 +24,7 @@
         <!-- @slot Custom input label -->
         <slot name="label" v-bind="{ label }">{{ label }}</slot>
       </label>
+      <!-- @slot if isPassword is true, show characters as dots in input -->
```

Comment:
@justyna-13 we can remove all `If` from `@slot` description. This slot can be described like 'Custom show password button' or something similar :)